sympy/tensor/tensor.py

```diff
@@ -2122,7 +2122,8 @@ def replace_with_arrays(self, replacement_dict, indices):
         replacement_dict
             dictionary containing the replacement rules for tensors.
         indices
-            the index order with respect to which the array is read.
+            the index order with respect to which the array is read. The
+            original index order will be used if no value is passed.
```

Comment:
This should have an example in the docstring

Reply:
Thank you for reviewing! I will finish it asap.